common/lib/capa/capa/responsetypes.py

```diff
@@ -1097,6 +1097,9 @@ def get_score(self, student_answers):
                 cmap.set(aid, 'correct')
             else:
                 cmap.set(aid, 'incorrect')
+            answer_variable = self.get_student_answer_variable_name(student_answers, aid)
```

Comment:
This does not seem like the most appropriate place for this to me. Can we move this to `evaluate_answers()`?